PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -2016,6 +2016,89 @@ class PurchasesTests: XCTestCase {
         verifyUpdatedCaches(newAppUserID: identityManager.currentAppUserID)
     }
 
```

Comment:
since this file is pretty big, we added the tests for subscriber attributes stuff to a separate one, https://github.com/RevenueCat/purchases-ios/blob/main/PurchasesTests/SubscriberAttributes/PurchasesSubscriberAttributesTests.swift

We should move these to that file since they're testing related stuff

Reply:
✅